src/api/data.js

```diff
@@ -15,7 +14,6 @@ function* signOut(key) {
     yield auth.signOut();
     return true;
   } catch (e) {
-    console.log('fail to login', e);
     return false;
```

Comment:
I'm curious why we decided to take out the console log? Was it because of the linter? I guess we should come up with some sort of error handling. That could be a chore for later.